test_cases/reverse_address.json

```diff
@@ -0,0 +1,166 @@
+{
+  "name": "reverse non-coarse",
+  "priorityThresh": 1,
+  "endpoint": "reverse",
+  "tests": [
+    {
+      "id": 1,
+      "status": "pass",
+      "user": "trescube",
+      "description": "there should be no non-address results, lat/lon is for Lancaster, PA",
+      "type": "dev",
+      "in": {
+        "point.lat": 40.038123,
+        "point.lon": -76.304096
+      },
+      "expected": {
+        "properties": [
+          {
+            "layer": "address"
+          }
+        ]
+      },
+      "unexpected": {
+        "properties": [
+          {
+            "layer": "locality"
+          },
+          {
+            "layer": "localadmin"
+          }
+        ]
+      }
+    },
+    {
+      "id": 2,
+      "status": "pass",
+      "user": "trescube",
+      "description": "there should be no non-address results, lat/lon is for Ventura County, CA",
+      "type": "dev",
+      "in": {
+        "point.lat": 34.35755,
+        "point.lon": -119.126008
+      },
+      "expected": {
+        "properties": [
+          {
+            "layer": "street"
+          }
+        ]
+      },
+      "unexpected": {
+        "properties": [
+          {
+            "layer": "county"
+          }
+        ]
+      }
+    },
+    {
+      "id": 3,
+      "status": "pass",
+      "user": "trescube",
+      "description": "addresses should be returned before coarse results, this lat/lon is for Red Lion, PA",
+      "type": "dev",
+      "in": {
+        "point.lat": 39.898111,
+        "point.lon": -76.607958
+      },
+      "expected": {
+        "properties": [
+          {
+            "layer": "address"
+          }
+        ]
+      },
+      "unexpected": {
+        "properties": [
+          {
+            "gid": "whosonfirst:locality:101717221"
```

Comment:
Yeah, I think its only in the `unexpected` case where using `gid` would be bad. If the record is superseded and in the same build we break something, we'd never know.